packages/yum/centos-8/Dockerfile

```diff
@@ -26,4 +26,7 @@ RUN \
     ruby \
     tar \
     zlib-devel && \
+  dnf install -y ${quiet} https://packages.groonga.org/centos/8/groonga-release-latest.noarch.rpm && \
```

Comment:
Can we merge this into `dnf install -y ${quiet} epel-release 'dnf-command(config-manager)'`?